LoRaEngine/test/LoRaWan.IntegrationTest/LoRaWan.IntegrationTest.csproj

```diff
@@ -41,6 +41,10 @@
   <ItemGroup>
     <AdditionalFiles Include="../../../stylecop.json" Link="stylecop.json" />
   </ItemGroup>
+  <ItemGroup>
+    <Compile Include="..\..\modules\LoRaWanNetworkSrvModule\LoRaWan.NetworkServer\ApiVersion.cs" Link="ApiVersion.cs" />
+    <Compile Include="..\..\modules\LoRaWanNetworkSrvModule\LoRaWan.NetworkServer\ServiceFacadeHttpClientHandler.cs" Link="ServiceFacadeHttpClientHandler.cs" />
```

Comment:
I see why you renamed the namespace now. We should move all constants and shared code to the LoraTools and rename it to Shared/Common.
That is a subject for a new PR though. :)